src/components/Footer/Footer.tsx

```diff
@@ -0,0 +1,141 @@
+import React from "react";
+import { Logo } from "../Logo/Logo";
+import depHealthHumanServicesWhite from "../../assets/img/logos/dept_health_human_services_usa_white.svg";
+import depHealthHumanServicesBlack from "../../assets/img/logos/dept_health_human_services_usa_black.svg";
+import medicaidLogoWhite from "../../assets/img/logos/medicaid_gov_white.svg";
+import medicaidLogoBlack from "../../assets/img/logos/medicaid_gov_black.svg";
+import { Icon, IconChoice } from "../Icon/Icon";
+type IntrinsicElements = JSX.IntrinsicElements["footer"];
+
+interface Link {
+  href?: string;
+  iconName?: IconChoice;
+  onClick?: () => any;
+  text: string;
+}
+
```

Comment:
I see JP is having an incoming interface thats similar maybe we can export one of them